docs/source/trainer.rst

```diff
@@ -157,6 +157,51 @@ Once you're done training, feel free to run the test set!
 
     trainer.test(test_dataloaders=test_dataloader)
 
+Saving test predictions is simple ! You can provide directly a tensor or list of dictionaries.
+
+.. code-block:: python
+
+    import pandas as pd
+
+    class MyModel(LightningModule):
+
+        def test_step(self, batch, batch_idx, dataloader_idx=None):
+            x, y = batch
+            output = self.layer(x)
+            loss = self.loss(y, output)
+
+            # Add directly a tensor
+            self.add_predictions(output)
+
+            return {"y": loss}
+
+    results = trainer.test(test_dataloaders=test_dataloaders)
+    for dataloader_idx, dataloader_result in enumerate(results_:
+
+        df = pd.json_normalize(dataloader_result["predictions"], sep='_')
```

Comment:
The fact that we use ` pd.json_normalize` tells me that the predictions have a non-trivial from (not just a tensor). Could we add a description what format the predictions have (list of dicts?)

Reply:
Done !